pkg/activator/net/throttler.go

```diff
@@ -251,24 +251,75 @@ func (rt *revisionThrottler) updateThrottleState(throttler *Throttler, backendCo
 	}
 }
 
+// pickIndices picks the indices for the slicing.
+func pickIndices(lt, selfIndex, numActivators int) (bi, ei int) {
```

Comment:
Can we pick speaking names for `lt`, `bi` and `ei`?

Reply:
len(trackers), b(egin)i(ndex), etc :)
but sure, I can make names slightly more expressive